Make modal-body wider so scroll bars are not truncated

redmunds opened the pull request:
This is for internal issue 366 and https://github.com/adobe/brackets-edge-web-fonts/issues/158.

redmunds committed:
make modal-body wider so scroll bars are not truncated

couzteau commented:
Hey @redmunds, I can confirm that the scrollbars are visible now, but there are 2 other issues with the EWF dialog:
- dialog witdh
- clipping and positioning

I have updated bug #366 in the edge code repo.

redmunds committed:
fix dialog width and positioning

redmunds commented:
Fixed remaining issues.

For dialog width, I added `width: 535px;` to `.edge-web-fonts-browse-dialog` rule to override `width: 600px;` in `.modal` base rule. Also had to remove `width: 520px;` from `.modal-header` which is not necessary.

For clipping and positioning, I removed `margin-top: -275px;` from `.edge-web-fonts-browse-dialog`. Modal dialog now automatically centers dialogs vertically, so this is no longer necessary.

Ready for review.

couzteau commented on styles/ewf-brackets.css:
```diff
@@ -28,9 +28,8 @@
-.edge-web-fonts-browse-dialog .modal-header {
-  width: 505px;
+.edge-web-fonts-browse-dialog .modal-body {
+    width: 520px;
```
consolidate with rule below? Both are for .edge-web-fonts-browse-dialog .modal-body

redmunds replied:
Good catch! Fixed.

couzteau commented:
Dialog looks good now. Done with review.

redmunds committed:
merge rules

redmunds commented:
Ready for re-review.

couzteau commented:
Merging.

couzteau commented:
Now  we need to update the SHA for EWF. Will set up a pull request
